scripts/qa_crowbarsetup.sh

```diff
@@ -2536,6 +2537,16 @@ function oncontroller_testsetup()
     . .openrc
 
     export LC_ALL=C
+
+    # prepare test image with the -test packages containing functional tests
+    if iscloudver 6plus; then
+        local mount_dir="/tmp/Cloud-Testing"
```

Comment:
> /var/lib/openstack-functional-tests perhaps ?

It's just containing packages. And maybe also requirements for the -test packages. So /var/lib/openstack-functional-tests sounds wrong to me